src/main/resources/entityResources/localization/eng/EntityMod-Card-Strings.json

```
@@ -79,6 +79,10 @@
     "NAME": "Tu",
     "DESCRIPTION": "Gain !B! Block. If you have entity:Ka, entity:Chant entity:Ka."
   },
+  "entity:VileWeave": {
+    "NAME": "Vile Weave",
+    "DESCRIPTION": "Gain !B! Block. Gain !entity:F! entity:Flux.",
```

Forgot that we need to change this to the selfMagicNumber instead of flux variable.
```suggestion
    "DESCRIPTION": "Gain !B! Block. Gain !entity:S! entity:Flux.",
```